packages/components/src/components/Table/Table.types.tsx

```diff
@@ -37,4 +37,8 @@ export type TableRowProps = {
   selected?: boolean;
   /**Spesifiserer om raden skal ha hover styling. Brukes hovedsakelig på klikkbare rader. */
   hoverable?: boolean;
+  /**Spesifiserer om raden med type="head" skal ha hvit bakgrunnsfarge. */
+  useWhiteBackground?: boolean;
+  /**Spesifiserer om radene skal ha motsatt even-odd bakgrunnsfarge sammenlignet med default. */
+  invertEvenOdd?: boolean;
```

Comment:
Tenker disse burde utledes ut i fra `density` på tabellen og skjules for konsumenter 🤔 Vi ønsker helst ikke at disse stilene skal brukes med andre densities enn `'extra-compact'`/`'alternative'` 😅

Reply:
Fixed.